packages/eslint-plugin/src/config/react.ts

```diff
@@ -104,4 +104,7 @@ const config: Linter.Config = {
     ]
 };
 
+// Using TypeScript "export" keyword until ESLint support ESM.
+// Otherwise we must deal with a weird CommonJS output from esbuild which is not worth it.
+// For more info, see: https://github.com/evanw/esbuild/issues/1079
```

Comment:
Sorry can't tag the actual line, so i'm putting the comment here

The imports in this file arent like in the other files

```
import type { Linter } from "eslint";
import { sourceFiles } from "../utils/patterns";
```

Reply:
I haven't updated any import. The only thing I did was move the file from `/lib` to `/src`. This is your original code :)

Reply:
in multiple file, you modified the imports, so i'm wondering why it hasnt been done in all files. For instance https://github.com/workleap/wl-web-configs/pull/69/files/d9e912a279a12486ad3ad271b9cdf3a6b584054b#r1206809947

Reply:
Resolved in DM